crypto/codec/cmt.go

```diff
@@ -21,9 +21,13 @@ func FromCmtProtoPublicKey(protoPk cmtprotocrypto.PublicKey) (cryptotypes.PubKey
 			Key: protoPk.Ed25519,
 		}, nil
 	case *cmtprotocrypto.PublicKey_Secp256K1:
-		return &secp256k1.PubKey{
+		return &secp256k1.PubKeyOld{
 			Key: protoPk.Secp256K1,
 		}, nil
```

Comment:
You should remove this case, for safety (check what @Raneet10 did in the peppermint rebase PR)

Reply:
Thanks, addressed here e3a7a8e305e1fe06f74f57eb7257c246e29130d7

Reply:
Resolved